benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+    return gen_grid_cross_rle(n_rows, n_cols, gen_int64), ColumnType.INT64
+
+
+def gen_dec_cross_rle_opr_dbl(n_rows, n_cols):
+    return gen_grid_cross_rle(n_rows, n_cols, gen_dbl), ColumnType.DOUBLE
+
+
+def gen_dec_cross_rle_opr_str(n_rows, n_cols):
+    return gen_grid_cross_rle(n_rows, n_cols, gen_str), ColumnType.FLS_STR
+
+
+def generate_all_encodings():
```

Ruff (0.11.9)

Missing return type annotation for public function `generate_all_encodings`

(ANN201)